frontends/precinct-scanner/src/api/config.ts

```diff
@@ -249,6 +254,12 @@ export function addTemplates(pkg: BallotPackage): AddTemplatesEvents {
         });
       }
 
+      // Set the precinct if there is only one precinct
+      if (pkg.electionDefinition.election.precincts.length === 1) {
+        const onlyPrecinct = pkg.electionDefinition.election.precincts[0];
+        await setPrecinctSelection(singlePrecinctSelectionFor(onlyPrecinct.id));
+      }
```

Comment:
How did you choose to add this logic to `addTemplates`? It seems a bit unrelated to the templates logic.

A few alternative ideas:
- Rename this function to better represent what it's doing holistically
- Move this logic to somewhere else (maybe where `addTemplates` is called)
- Move this logic to the backend (since that's the general direction we're moving in)

Reply:
I chose to add it to to `addTemplates` because it's also where the election definition is set, so it seemed appropriate to set the precinct here as well. Originally (in the first commit) I had it where `addTemplates` is called. I don't think it was much clearer, and it was way harder to test. `addTemplates` should probably be renamed.

I prefer the idea of moving it to the backend, I'll look at that.